src/main/java/edu/wpi/first/shuffleboard/WidgetPaneController.java

```diff
@@ -241,4 +303,104 @@ private Menu createChangeMenus(WidgetTile tile) {
     return changeView;
   }
 
+  private void moveTileLeft(WidgetTile tile, int count) {
+    for (int i = 0; i < count; i++) {
+      Optional<Runnable> move = moveTileLeft(tile);
+      if (move.isPresent()) {
+        move.get().run();
+      } else {
+        // Can't move any further, bail
+        break;
+      }
+    }
+  }
+
+  private Optional<Runnable> moveTileLeft(WidgetTile tile) {
+    TileLayout layout = pane.getTileLayout(tile);
+    final int previousColumn = layout.origin.col - 1;
+    if (!pane.isOverlapping(previousColumn, layout.origin.row,
+        layout.size.getWidth(), layout.size.getHeight(), n -> tile == n) && previousColumn >= 0) {
```

Comment:
can we make this just layout.size? and can we use previousPosition instead of previousColumn, layout.origin.row?